docs/topics/proc_configure-netty-logging-vertx.adoc

```diff
@@ -0,0 +1,20 @@
+
+[#_configure-netty-logging-vertx]
+= Configuring Netty Logging
+
+Netty does not rely on an external logging configuration using system properties.
```

Comment:
Can you give some context for what Netty is or why its important? I cannot find any other references to Netty in the current version of the Vert.x Guide.

Reply:
Thanks for catching this. I meant to post a link to Netty docs in this section, but somehow I failed to do so.
Anyway, I'll add the link and some text explaining that Netty is a library implemented by VertX to manage asynchronous network communication in applications.

Reply:
Netty is a library _used_ by Vert.x.

Reply:
@Ladicek thank you:), my previous wording was indeed incorrect